### ISSUE TITLE
Config and auto-dl

### ISSUE THREAD

**Issue by StrangeDOS** [Xonshiz/comic-dl#77]
Tried out the config file.  Could not get it to work.

1. Created a folder with just one .cbz file (so there would be stuff missing that needed to be download)
2. Created a config file (attached).  Included a title/url associated with the folder above
3. Executed 'comic_dl --auto-download -v'

7 seconds, then nothing....
C:\Program Files\nodejs>comic_dl.exe --auto-download -v

***Starting the script in Verbose Mode***

Downloading Full!
Finished Downloading
Total Time Taken To Complete : 7.59099984169

Attaching log and config file.  
[Error_Log.log](https://github.com/Xonshiz/comic-dl/files/1624955/Error_Log.log)

**Comment by Xonshiz** [comment]
I don't see any config file. Could you upload on pastebin or the file here again?

**Comment by StrangeDOS** [comment]
Sorry about that.  The site doesn't allow that file type to be attached.  I just didn't notice the message when I tried to send it last night.  Renamed the file to .txt to get it uploaded.

[comic_dl_config.config.txt](https://github.com/Xonshiz/comic-dl/files/1626665/comic_dl_config.config.txt)

**Comment by StrangeDOS** [comment]
I tried using cbz instead of .cbz, and no help.  attaching log and config files
[comic_dl_config.config.txt](https://github.com/Xonshiz/comic-dl/files/1627588/comic_dl_config.config.txt)
[Error_Log.log](https://github.com/Xonshiz/comic-dl/files/1627589/Error_Log.log)

Looks like it is putting together all the URLs but then just stops. 

DEBUG: All Links : ['Action-Comics-2016/Issue-996', 'Action-Comics-2016/Issue-957?id=66214', 'Action-Comics-2016/Issue-958?id=66215', 'Action-Comics-2016/Issue-959?id=74470', 'Action-Comics-2016/Issue-960?id=78052', 'Action-Comics-2016/Issue-961?id=81540', 'Action-Comics-2016/Issue-962?id=84623', 'Action-Comics-2016/Issue-963?id=87661', 'Action-Comics-2016/Issue-964?id=90642', 'Action-Comics-2016/Issue-965?id=92290', 'Action-Comics-2016/Issue-966?id=93968', 'Action-Comics-2016/Issue-967?id=95158', 'Action-Comics-2016/Issue-968?id=96216', 'Action-Comics-2016/Issue-969?id=97940', 'Action-Comics-2016/Issue-970?id=99312', 'Action-Comics-2016/Issue-971?id=100540', 'Action-Comics-2016/Issue-972?id=102176', 'Action-Comics-2016/Issue-973?id=103792', 'Action-Comics-2016/Issue-974?id=105181', 'Action-Comics-2016/Issue-975?id=106290', 'Action-Comics-2016/Issue-976?id=107853', 'Action-Comics-2016/Issue-977?id=109695', 'Action-Comics-2016/Issue-978?id=110643', 'Action-Comics-2016/Issue-979?id=111789', 'Action-Comics-2016/Issue-980?id=112920', 'Action-Comics-2016/Issue-981?id=114639', 'Action-Comics-2016/Issue-982?id=115953', 'Action-Comics-2016/Issue-983?id=116877', 'Action-Comics-2016/Issue-984?id=117952', 'Action-Comics-2016/Issue-985?id=118779', 'Action-Comics-2016/Issue-986?id=119926', 'Action-Comics-2016/Issue-987?id=121377', 'Action-Comics-2016/Issue-988?id=122304', 'Action-Comics-2016/Issue-989?id=123071', 'Action-Comics-2016/Issue-990?id=123967', 'Action-Comics-2016/Issue-991?id=124772', 'Action-Comics-2016/Issue-992?id=125612', 'Action-Comics-2016/Issue-993?id=126574', 'Action-Comics-2016/Issue-994?id=127530', 'Action-Comics-2016/Issue-995?id=128515']

**Comment by Xonshiz** [comment]
Thank you for all this @StrangeDOS . I'll have to check this on my end now.

**Comment by Xonshiz** [comment]
Okay, about this issue, clear one thing please. Have you downloaded at least 1 chapter/issue for this comic?I mean, does that directory have some chapters downloaded already? Plus, it would also be great if you could post a screenshot of the directoty.

**Comment by StrangeDOS** [comment]
Okay.  Sorry it took me a while to respond.  I got to thinking after your last message that maybe I didn't really understand how it was supposed to work.  So I went back and read your instructions again.  That's when I started to see where I got it wrong.

I got the whole thing with the directory names mixed up.  I thought that the prompt for the location to store the comic series was asking for where I wanted the directory to be created....NOT the directory name of the actual directory.  

In other words, when prompted, I was giving it...
F:\DownloadComic
...instead of....
F:\DownloadComic\New Super Man

Which matched my default directory....so BLAMO!  Once I realized the mistake, I actually got it working...sort of.  I ran into a second issue unrelated to the error but still a problem with the config file.  

I'll submit a new bug.  Close this one as 'user error'.

**Comment by rawdpiper** [comment]
Ran the auto-dl function but the manga chapters were not updated
@StrangeDOS @Xonshiz 
[Error_Log.log](https://github.com/Xonshiz/comic-dl/files/1631691/Error_Log.log)

**Comment by Xonshiz** [comment]
@rawdpiper could you please open a new issue with the proper format? Like the comman you provided and then this error log?
